package.json

```diff
@@ -46,15 +45,15 @@
         "eslint-plugin-prettier": "^3.1.0",
         "prettier": "^1.18.2",
         "rimraf": "^2.6.3",
-        "wait-on": "^3.2.0"
+        "wait-on": "^3.2.0",
+        "typescript": "^3.8.0-dev.20191203"
```

Comment:
Thanks for explaining `devDependencies` in person.

I forgot why we said its ok to pin this specific version and how to keep it in sync with eden?

Reply:
I don't really need to pin this version, it's just in the event that someone changes this library and recompiles, that they're going to be using the same version that eden is.